storage/src/state/ledger.rs

```diff
@@ -94,12 +94,14 @@ pub struct LedgerState<N: Network> {
 impl<N: Network> LedgerState<N> {
     ///
     /// Opens a new writable instance of `LedgerState` from the given storage path.
+    /// The `validation_increment` parameter determines the number of blocks to be
+    /// handled during the incremental validation process.
     /// For a read-only instance of `LedgerState`, use `LedgerState::open_reader`.
     ///
     /// A writable instance of `LedgerState` possesses full functionality, whereas
     /// a read-only instance of `LedgerState` may only call immutable methods.
     ///
-    pub fn open_writer<S: Storage, P: AsRef<Path>>(path: P) -> Result<Self> {
+    pub fn open_writer<S: Storage, P: AsRef<Path>>(path: P, validation_increment: u32) -> Result<Self> {
```

Comment:
This interface is not intuitive.

One option is to make it a parameter in Environment, otherwise it should be removed.

Reply:
There needs to be a way of parametrizing it, otherwise it can't be tested; in order for the interface to be more intuitive, there can be another function, perhaps a `#[doc(hidden)]` one, that gets called by `open_writer`; I'll propose it in the next commit.

Reply:
Thank you for the change, it looks great